From a97d6e2b13146783831b166ec5e8b33b29c514b0 Mon Sep 17 00:00:00 2001
From: Dusty Mabe <dusty@dustymabe.com>
Date: Fri, 25 Jun 2021 15:27:37 -0400
Subject: [PATCH] fix(network-manager): pull in network.target in
 nm-initrd.service

Otherwise units with `After=network.target` won't have any effect.
---
 modules.d/35network-manager/nm-initrd.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/35network-manager/nm-initrd.service b/modules.d/35network-manager/nm-initrd.service
index 815797a4..5f9b9043 100644
--- a/modules.d/35network-manager/nm-initrd.service
+++ b/modules.d/35network-manager/nm-initrd.service
@@ -3,6 +3,7 @@ DefaultDependencies=no
 Wants=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=dracut-cmdline.service
+Wants=network.target
 Before=network.target
 ConditionPathExists=/run/NetworkManager/initrd/neednet
 ConditionPathExistsGlob=|/usr/lib/NetworkManager/system-connections/*
-- 
2.31.1

